samples/MegaDrone/src/main/cpp/native-lib.cpp

```diff
@@ -18,8 +18,9 @@
 #include <string>
 #include <vector>
 
-#include "AudioEngine.h"
-#include "../../../../../src/common/OboeDebug.h"
+#include "PlayAudioEngine.h"
+
+PlayAudioEngine *engine; // This should be passed back as a long
```

Comment:
This file merge conflicted. I am trying to figure out the best way to merge in. The latest commit fixed the merge conflict but it looks like it will overwrite your changes, so I can restore back. Should I merge locally to resolve conflicts and then push directly to master? Or I can rebase locally to preserve history, push to this branch, then push to master (I am leaning towards this option).

Reply:
I'd do the following locally: 

> git checkout refactor-samples
> git rebase master
> **resolve conflicts**
> git push -f origin refactor-samples

Note the last -f is important because otherwise you'll get all the commits from master appearing in this pull request.

Reply:
It should merge cleanly, passing back pointers as before now.